### ISSUE TITLE
shuffle sound when starting a hand

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#541]
Play a shuffle sound when starting a hand.

This is a successor to "more sounds" #398. The game change was pretty important. A shuffle sound at the start of a hand was asked for by Crunch and as such, I don't want it to slip forever, however, it probably doesn't make sense to add this before adding persistent player preferences (#540), because my guess is most people won't want it.

Here's what needs to be done:

- [x] change audio preference representation from an enum to a wrapped i16
- [x] create a migration to convert from the old representation to the new representation
- [x] implement SelectHtmlExt trait for the new representation
- [x] rework the bell icon code to handle a subset of changes
- [x] add and stylize the checkboxes for the six sounds
- [ ] modify the code to play the sounds only when appropriate:
  - [x] New Game
  - [x] Your Turn
  - [x] Running Late
  - [x] Player Bust
  - [ ] New Tournament
  - [x] New Hand

**Comment by ctm** [comment]
I need to poke around the various CC0 sound sites and dig up contact info for the CC0 sounds I'm already using. No only have I added persistent player prefs (#540), but the way they're implemented makes adding new prefs fairly easy, so this really is trivial.

**Comment by ctm** [comment]
OK, it's not trivial if I choose to refactor the sound preference into a bunch of checkboxes. It's still easy enough for me to do, but I might not have it ready in time for this evening's game.

**Comment by ctm** [comment]
This morning (six hours ago!), when I did my morning deploy I encountered some log lines that have kept me busy. The good news is that yesterday I got the layout for the checkboxes in the preferences modal done. However, with so much of today gone already, there's now a strong chance that I won't get this done by this evening.

OTOH, I did pick up a shuffle sound and did get my [CC0 thanks table](https://ctm.github.io/docs/players_manual/thanks/mb2.html#cc0-creators) mostly populated.
